lib/kochiku/worker.rb

```diff
@@ -60,6 +60,32 @@ def msg2str(msg)
       def build_strategy
         @build_strategy ||= BuildStrategyFactory.get_strategy(settings.build_strategy)
       end
+
+      def instance_type
+        @instance_type ||= begin
+          path = File.join(ENV["HOME"], 'instance_type.txt')
```

Comment:
I don't like dumping stuff in `$HOME`, we should keep it inside of the kochiku-worker dir. Either kochiku-worker root or `kochiku-worker/tmp/`